hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/TokensConfig.java

```diff
@@ -32,6 +32,7 @@ public record TokensConfig(
         @ConfigProperty(defaultValue = "10") @NetworkProperty int maxCustomFeesAllowed,
         @ConfigProperty(defaultValue = "2") @NetworkProperty int maxCustomFeeDepth,
         @ConfigProperty(defaultValue = "1000") @NetworkProperty long maxRelsPerInfoQuery,
+        @ConfigProperty(value = "reject.enabled", defaultValue = "true") @NetworkProperty boolean tokenRejectEnabled,
```

Comment:
Why do we need this config property. Since this is a new transaction, if we don't  set it in throttles or provide no permissions, this can be disabled.

Reply:
The docs specified that we would need a feature toggle for TokenReject. - [Reference here](https://github.com/hashgraph/hedera-services/pull/12786/files#diff-345e91521530c2849705b8d557dc66a644189322a8cc02910e1b5ea11cd73929R24)

Reply:
Doesn't hurt, but its not needed for new transaction types